specs/phase1/fraud-proofs.md

```diff
@@ -29,42 +32,259 @@ This document describes the shard transition function and fraud proofs as part o
 
 ## Fraud proofs
 
+### Shard state transition function
+
+```python
+def shard_state_transition(beacon_state: BeaconState,
```

Comment:
The naming of `shard_state_transition` is pretty similar to `get_shard_transition`, but I can't think of a better name now...